src/devices/Ds1307/samples/Ds1307.Samples.csproj

```diff
@@ -6,7 +6,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Device.Gpio" Version="$(SystemDeviceGpioPackageVersion)" />
+    <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj">
+      <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
```

Comment:
I'm still confused why is `RuntimeIdentifier=linux` needed

Reply:
I logged an issue to fix that in the future: #623